feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/post/PostItem.kt

```diff
@@ -99,6 +106,7 @@ internal fun PostItem(
         PostItemViewType.ThreadContextPostItem -> true
         else -> false
     }
+    val isPinned = post is IStandalonePost && post.displayPriority == DisplayPriority.PINNED
```

Comment:
Is the condition `post is IStandalonePost` needed here? I feel like we should display the post as pinned according to the data status and the restriction that only standalone posts _can_ be pinned should be handled by the Server and the bottomSheet UI

Reply:
Ah, that makes sense. Then it's fine for me how it is 👍